### ISSUE TITLE
[Feature Request]: Create a "Try Apache Beam YAML" landing page

### ISSUE THREAD

**Issue by svetakvsundhar** [apache/beam#27450]
### What would you like to happen?

On [this page](https://beam.apache.org/get-started/try-apache-beam/), we should have a section for the YAML colab notebook once https://github.com/apache/beam/pull/27284 is merged.

### Issue Priority

Priority: 3 (nice-to-have improvement)

### Issue Components

- [ ] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [X] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

**Comment by svetakvsundhar** [comment]
cc: @bzablocki
